module1.js

```diff
@@ -111,6 +136,10 @@ function question7(array1, array2, extraElement) {
 
 function question8(retailPrice, taxAmount) {
   // TODO:
+  var ret = retailPrice.toCurrency();
```

Comment:
Strings do not have a `toCurrency` method! The error message `TypeError: retailPrice.toCurrency is not a function` tells you where the issue is. You do need to _parse_ the input, though, to extract the numerical part.